modules/nf-core/diamond/blastp/meta.yml

```diff
@@ -77,6 +82,8 @@ output:
 authors:
   - "@spficklin"
   - "@jfy133"
+  - "@vagkaratzas"
```

Comment:
```suggestion
```
This field is for who originally wrote the module 🤷🏻‍♂️ 

https://github.com/nf-core/modules/pull/4173#issuecomment-1771271609

Reply:
Done